javascript/react/src/components/TableTransform.types.ts

```diff
@@ -83,7 +83,5 @@ export interface TableTransformProps {
 
 	onDelete?: (index: number) => void
 
-	onPreview?: (name: string) => void
-
 	onDuplicate?: (step: Step) => void
```

Comment:
Is this still necessary?

Reply:
the onDuplicate? We're not using it now, should we remove it for good?

Reply:
Well, I see a couple of options to consider:
* Keep it as is, and ensure the functionality stays up-to-date to correctly duplicate the step. It would make sense to do the same with `onPreview` instead of removing it - but call it `onSelect` instead, which would give a consistent API to selecting tables.
* Remove it to simplify the API, and only have `onChange` and `onDelete` callbacks for the steps

I'm a little partial to option 2 because it means a simpler API for functionality that I don't think we'll need. One approach to getting the functionality back if it (or other functions) are desired would be to accept a command bar prop that replaces the default

Reply:
FWIW, I think the direct manipulation interface removes the need for step duplication - it's much more fluid, so I don't see why we'd need the duplicate button here. With an eventual advanced/graph editor maybe, but that'll be a different component...

Reply:
Yeah I think getting it back will be simple in case we need it, and clean to remove it for now